spec/classes/snmp_init_spec.rb

```diff
@@ -607,7 +607,7 @@
           ).that_requires('Package[snmpd]').that_notifies('Service[snmpd]')
         }
         case facts[:os]['release']['major']
-        when '8', '9', '10', '16.04', '18.04'
+        when '8', '9', '10', '16.04', '18.04', '20.04'
```

Comment:
This is effectively all supported Debian / Ubuntu versions. I'm not sure the else makes sense anymore.

Reply:
True though they are just following the pattern that is already there and should not be expected to improve that as part of their PR to add support for a new platform.